app/routes/jokes/new.tsx

```diff
@@ -7,37 +7,25 @@ import {
   useCatch,
   useTransition,
 } from "@remix-run/react";
+import type { ErrorResult, UnpackData } from "domain-functions";
+import { inputFromFormData } from "domain-functions";
+import { inputFromForm } from "domain-functions";
 
 import { JokeDisplay } from "~/components/joke";
-import { db } from "~/utils/db.server";
-import { requireUserId, getUserId } from "~/utils/session.server";
+import { getUserId } from "~/utils/session.server";
+import { createJoke, jokeSchema } from "~/domains/jokes";
+import { enforceUser } from "~/domains/user";
+import { fieldHasErrors, fieldFirstMessage } from "~/utils/helpers";
 
 export const loader: LoaderFunction = async ({ request }) => {
-  const userId = await getUserId(request);
-  if (!userId) {
+  const result = await enforceUser({ id: await getUserId(request) });
```

Comment:
why is `enforceUser` definition empty ? I would use `getUserId` straight or create a helper such as `requireUser` (that would not be a DF either).

Reply:
`enforceUser` is defined at domain functions and is used as a type on line 25

Reply:
Sure, but the function definition is empty, so it's just a bit odd. But no big deal